packages/mermaid/src/rendering-util/createText.js

```diff
@@ -76,6 +76,22 @@ function createTspan(textElement, lineIndex, lineHeight) {
     .attr('dy', lineHeight + 'em');
 }
 
+/**
+ * Compute the width of rendered text
+ * @param {object} parentNode
+ * @param {number} lineHeight
+ * @param {string} text
+ * @returns {number}
+ */
+function testWidthOfText(parentNode, lineHeight, text) {
+  const testElement = parentNode.append('text').attr('y', '-10.1');
```

Comment:
Where does the `-10.1` value come from? 
We should avoid [Magic Numbers](https://en.wikipedia.org/wiki/Magic_number_(programming))

Reply:
Certainly.  As it turns out this y offset isn't necessary as this node is disposed later anyway - and the y value appears to not have an effect on the text width value.